### DISCUSSION TITLE
Data format for displaying metrics

### DISCUSSION THREAD

**Discussion by sinusen** [grafana/grafana-json-datasource#140] [Q&A]
![image](https://user-images.githubusercontent.com/70787000/123906540-b4c53e00-d9b7-11eb-9d50-70f377d92734.png)
I'm looking to display to display this graph using JSON API. With PostgreSQL, I could select one column as metric and it used to display data of different metrics in different colors
```
SELECT
  to_timestamp(epoch_ms/1000)) as time,
  COALESCE(b.abbr,a.fg_description) AS metric,
  variance/bt_qty*100 as yield
```
However for the JSON API, I am not able to get this to work. I have configured it like in the below screenshot
![image](https://user-images.githubusercontent.com/70787000/123907014-8eec6900-d9b8-11eb-963d-50718a5ed612.png)
It don't respect the metric. I even tried setting the Metric in the experimental feature to my metric property. The structure of my response is as follows
```
 {
      "items":[{
                        "time":"Tue Jun 29 2021 00:00:00 GMT+1000 (Australian Eastern Standard Time)", 
                         "metric":"DOMCM",
                          "yield":"3.993560588667439"
                    }]
}
```
In short, what should be the structure of my response to show multiple series in a time series graph with series name as legend?
